main.py

```diff
@@ -0,0 +1,41 @@
+import pygame
+from minefield import *
+
+pygame.init()
```

Comment:
All of your main code needs to exist in a main() function. You can read about this convention